cmd/volume.go

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2017 XLAB, Ltd.
+ *
+ * This work is open source software, licensed under the terms of the
+ * BSD license as described in the LICENSE file in the top-level directory.
+ */
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/mikelangelo-project/capstan/hypervisor"
+	"github.com/mikelangelo-project/capstan/hypervisor/qemu"
+)
+
+// Volume is an extended version of hypervisor.Volume with volume creation data.
+type Volume struct {
+	hypervisor.Volume
+	SizeMB int64
+	Name   string
+}
+
+const VOLUMES_DIR string = "volumes"
+
+// CreateVolume creates volume with given specifications.
+func CreateVolume(packagePath string, volume Volume) error {
```

Comment:
@miha-plesko how long does it take to create a volume of 100G (raw, qcow2)? If it's more than few seconds, print this info to the user.

Otherwise, as the workflow of volume creation is rather _simple_, I do not see a point in adding `verbose` flag. If something goes wrong, the user is informed through an error message.

Reply:
Okay, I agree. Considering instantaneous execution and as you say, simplicity of the workflow (which was indeed not so obvious to me at first), it makes sense not to include verbosity control.